src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -14,125 +14,152 @@ import SentryTypes from 'app/sentryTypes';
-    }
-    {...props}
-  />
-);
-
-SuggestedOwnerHovercard.propTypes = {
-  /**
-   * The suggested actor.
-   */
-  actor: PropTypes.oneOfType([
-    // eslint-disable-next-line react/no-typos
-    SentryTypes.User,
-    // Sentry user which has *not* been expanded
-    PropTypes.shape({
-      email: PropTypes.string.isRequired,
-      id: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-    // Unidentifier user (from commits)
-    PropTypes.shape({
-      email: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-    // Sentry team
-    PropTypes.shape({
-      id: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-  ]),
-
-  /**
-   * The list of commits the actor is suggested for. May be left blank if the
-   * actor is not suggested for commits.
-   */
-  commits: PropTypes.arrayOf(
-    PropTypes.shape({
-      message: PropTypes.string.isRequired,
-      dateCreated: PropTypes.string.isRequired,
-    })
-  ),
-
-  /**
-   * The list of ownership rules the actor is suggested for. Maybe left blank
-   * if the actor is not suggested based on ownership rules.
-   */
-  rules: PropTypes.arrayOf(PropTypes.array),
-};
+        }
+        body={
+          <HovercardBody>
+            {commits !== undefined && (
+              <React.Fragment>
+                <div className="divider">
+                  <h6>{t('Suspect Commits')}</h6>
+                </div>
+                <div>
+                  {commits
+                    .slice(0, this.state.commitsExpanded ? commits.length : 3)
+                    .map(({message, dateCreated}, i) => (
+                      <CommitReasonItem key={i}>
+                        <CommitIcon />
+                        <CommitMessage message={message} date={dateCreated} />
+                      </CommitReasonItem>
+                    ))}
+                </div>
+                {commits.length > 3 && !this.state.commitsExpanded ? (
+                  <ViewMoreButton
+                    onClick={() => this.setState({commitsExpanded: true})}
+                  />
+                ) : null}
+              </React.Fragment>
+            )}
+            {rules !== undefined && (
+              <React.Fragment>
+                <div className="divider">
+                  <h6>{t('Ownership Rules')}</h6>
+                </div>
+                <div>
+                  {rules
+                    .slice(0, this.state.rulesExpanded ? rules.length : 3)
+                    .map(([type, matched], i) => (
+                      <RuleReasonItem key={i}>
+                        <OwnershipTag tagType={type} />
+                        <OwnershipValue>{matched}</OwnershipValue>
+                      </RuleReasonItem>
+                    ))}
+                </div>
+                {rules.length > 3 && !this.state.rulesExpanded ? (
+                  <ViewMoreButton onClick={() => this.setState({rulesExpanded: true})} />
```

Comment:
Personally, I prefer seeing this more explicitly laid out (unless it's re-used a few times):

```
<ViewMoreButton>
  <ViewMoreLink onClick={}>View More</ViewMoreLink>
</ViewMoreButton>
```

Otherwise we have to jump around a bit between the render and the styled component. SCs should mostly be just css.

Reply:
Fixed with other changes